array/chunk.js

```diff
@@ -0,0 +1,6 @@
+export default size => xs =>
```

Comment:
Please use `count` vs `size` to keep the naming consistent with previously implemented helpers (`range` etc.).